coverage-tests/configuration/eg_configuration.js

```diff
@@ -0,0 +1,11 @@
+module.exports = {
+    name: 'eyes_selenium_python',
+    emitter: 'https://raw.githubusercontent.com/applitools/sdk.coverage.tests/eg_python/python/emitter.js',
+    overrides: [
+        'https://raw.githubusercontent.com/applitools/sdk.coverage.tests/eg_python/python/overrides.js',
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/applitools/eyes.sdk.python/pullRequest?prid=7915070)
```suggestion
        "https://raw.githubusercontent.com/applitools/sdk.coverage.tests/eg_python/python/overrides.js",
```